### DISCUSSION TITLE
Github Pages Deployment not happening and Link not Getting generated

### DISCUSSION THREAD

**Discussion by saiakhil2012** [community/community#40883] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Suddenly changes to the branch linked to Github pages did not propagate to the webpage. I unpublished the website. And now, not able to publish the website again. The re-deployment of the webpage is not happening from github pages side

@github-staff Is Github Pages, new deployments of page feature down or not fully functional? Any suggestions on resolution for above mentioned problem? Thanks

(But According to [Status of Github](https://www.githubstatus.com/), everything is normal)

**Comment by radar-blip** [comment]
This one here? https://saiakhil2012.github.io/vmware-blockchain-samples/
Seems to be working now.

**Comment by saiakhil2012** [comment]
@linuxlawson No, I meant, this one actually --> https://vmware-samples.github.io/vmware-blockchain-samples/

**Reply by radar-blip** [reply]
Its working. 

![working](https://user-images.githubusercontent.com/39196767/205818408-5cd5e66c-2b28-46cc-9acd-7b7c0f4ed3fd.png)

**Comment by saiakhil2012** [comment]
Looks like there was some slowness or bug or outage in terms of deployment of changes to github pages and deployment of new github pages. Now, the github page is working again. It did not work for around 30 to 40 minutes. I also just noticed that its working now again, just before seeing your reply. Thanks @linuxlawson.

**Reply by radar-blip** [reply]
No problem. Just some kind of delay.

**Reply by saiakhil2012** [reply]
Yeah, looks like it. thanks a lot! But it was a very long delay compared to more general delay of just 2 to 3 mins.
